src/test/java/aquality/selenium/waitings/ConditionalWaitTests.java

```diff
@@ -97,14 +98,17 @@ public void testTimeoutExceptionShouldBeThrownIfDriverConditionIsNotMetAndDefaul
                     "Condition should be true");
 
         }catch (org.openqa.selenium.TimeoutException e){
-            double duration = timer.stop();
+            DurationSample durationSample = new DurationSample(timer.duration(), timeoutCondition, 7);
+            BrowserManager.getBrowser().quit();
```

Comment:
shouldn't we quit at AfterMethod?

Reply:
no, we should not. because waitFor creates Browser instance, but waitForTrue not.
probably we can divide the class into two: for waitFor and for waitForTrue. But we use the same timeout variables common to both groups of test methods. So, I think we can leave it as is.